tests/integration/test_solver_slippage.py

```diff
@@ -257,6 +260,45 @@ def test_similar_slippage_for_period(self):
             delta=delta,  # |v1 - v2| ~ 0.004  --> 4 USD (with ETH at 1000)
         )
 
+    def test_limit_order_slippage(self):
+        """This is an internal buffer trade containing a limit order (with a surplus_fee)"""
+        table_name = "incoming_and_outgoing"
+        v1_result, v2_result = self.get_cte_rows(
+            table_name,
+            period=AccountingPeriod("2022-11-29", 1),
+            tx_hash="0xfe4589525c1ed764273fbca9120b0e5f7f101d5d4996939ead95a50312f4d8b3",
+            v1_cache="01GKS1X2Y18ECYRRJPCSGEE57X",
+            v2_cache="01GKS1X8BPMXMD9FQ4T1ER22YW",
```

Comment:
I totally understand the intend of this. 
But isn't it also devaluing the test? Especially, since `exec_or_get` does not check whether the fetched query result the same parameters and the SQL query?

Reply:
These tests do no run in CI. So I think it's fine the way it is.